taxi/views.py

```diff
@@ -1,3 +1,18 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from taxi.models import Driver, Car, Manufacturer
+
 
 # Create your views here.
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
```

Comment:
It's a good practice to handle exceptions. What if the `Driver` model is empty? Consider using `Driver.objects.all().count()` and handle the case when there's no driver in the database.